components/Voorbeelden/404pagina.njk

```diff
@@ -0,0 +1,14 @@
+{% render '@header', {menu: menu, breadcrumbs: breadcrumbs} %}
+<main>
+  <div class="row">
+    <div class="col-md-8 col-md-offset-2">
+      <h1>404 Pagina niet gevonden</h1>
+      <p>We hebben de pagina niet gevonden. Dat kan komen doordat wij die hebben verplaatst of verwijderd, of doordat de ingevoerde URL niet klopt.</p>
+      <ul>
+        <li>Controleer de spelling van de pagina.</li>
+        <li>Ga terug naar de <a href="#">homepage.<a></li>
```

Comment:
De `.` wat het einde van de zin aangeeft staat meestal niet in de anchor van het laatste woord:

```html
<li>Ga terug naar de <a href="#">homepage<a>.</li>
```